Jenkinsfile

```diff
@@ -649,7 +649,20 @@ try {
   }
 
   stage('Integration Test') {
-    parallel 'Python GPU': {
+  	parallel 'Onnx CPU': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/it-onnx-cpu') {
+          init_git()
+          unpack_lib('cpu')
+          timeout(time: max_time, unit: 'MINUTES') {
+            sh "${docker_run} cpu --dockerbinary docker PYTHONPATH=./python/ pytest tests/python-pytest/onnx/onnx_backend_test.py"
+            sh "${docker_run} cpu --dockerbinary docker PYTHONPATH=./python/ pytest tests/python-pytest/onnx/test_onnx.py"
```

Comment:
inconsistent filenames

Reply:
@marcoabreu do you mean that either both should be ("test_onnx" and "test_onnx_backend") or ("onnx_backend_test" and "onnx_test")?